fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/JcrRdfTools.java

```diff
@@ -322,10 +322,17 @@ public void removeProperty(final FedoraResource resource,
 
         if (isManagedPredicate.apply(predicate)) {
 
-            throw new ServerManagedPropertyException("Could not remove triple containing predicate "
-                    + predicate.toString()
-                    + " to node "
-                    + node.getPath());
+            throw new ServerManagedPropertyException("Cannot update the container as the given RDF is "
+                        + "out-of-date based on the value of its fedora:lastModified triple. Predicate is "
```

Comment:
We should not hardcode `fedora:lastModifed` into this error message since the exception can be thrown for any server-managed property.
The existing message seems very correct. I would suggest leaving the message as-is.